web-client/src/presenter/computeds/caseDetailHeaderHelper.ts

```diff
@@ -1,8 +1,46 @@
 import { Case } from '../../../../shared/src/business/entities/cases/Case';
 import { ClientApplicationContext } from '@web-client/applicationContext';
 import { Get } from 'cerebral';
+import {
+  PRACTICE_TYPE,
+  ROLES,
+} from '@shared/business/entities/EntityConstants';
+import { RawIrsPractitioner } from '@shared/business/entities/IrsPractitioner';
+import { RawPractitioner } from '@shared/business/entities/Practitioner';
+import { RawUser } from '@shared/business/entities/User';
 import { state } from '@web-client/presenter/app.cerebral';
 
+const isUserADojPractitioner = (
+  user: RawUser | RawPractitioner | RawIrsPractitioner,
+): boolean => {
+  if (user.role !== ROLES.irsPractitioner) return false;
+  const irsPractitioner: RawPractitioner = user as RawPractitioner;
+  if (irsPractitioner.practiceType !== PRACTICE_TYPE.DOJ) return false;
+  return true;
+};
+
+const shouldShowRepresentAPartButton = (
```

Comment:
Missing a 'y' on 'Party' here 😄

Reply:
Hmmm I think there was a reason 'Y' I did that..... I will fix it.